test/e2e/save_test.go

```diff
@@ -224,13 +224,14 @@ default-docker:
 	})
 
 	It("podman save --multi-image-archive (untagged images)", func() {
+		Skip("FIXME: WTF IS THIS TEST SUPPOSED TO DO????")
```

Comment:
Someone please address this.

Reply:
This test calls `multiImageSave()` which contains the real test logic the call before just gets all image ids. I think the `Expect(len(RESTORE_IMAGES))...` can be removed or it should check that the len() >0

Reply:
Grrr. Github ate my response to this.

Short version: this was PEBKAC. I have to run with `NO_TEST_CACHE=1` because of my super-slow DSL. Hence `len(ids) == 0` in my environment, causing test to fail. I've rewritten the `Expect` in a more legible (I hope) way and resubmitted. Sorry to have wasted your time on this one.